### PR TITLE
[stable/concourse]fix typo in README.md

### PR DESCRIPTION
**Pull request by scf0920**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rjkernick** [comment]
Please fix merge conflict on chart.yaml

**Comment by william-tran** [comment]
Thanks for the fix @scf0920, please update the chart version so it can be merged.

File: stable/concourse/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 1.2.3
+version: 1.2.4
```
Comment (william-tran) [inline stable/concourse/Chart.yaml]: bump patch version 1.3.x

**Review by william-tran — CHANGES_REQUESTED** [review]

**Comment by scf0920** [comment]
@william-tran @rjkernick  Thanks. The version has already updated. Please continue merge 😄

**Comment by william-tran** [comment]
sorry @scf0920 this might need another bump, but your next in line!

**Comment by scf0920** [comment]
@william-tran The version update so fast...Now version has already updated. Please merge. 😀 Thanks

**Pull request merged**